### PR TITLE
feat(model): add Book model with name,desc,release

### PR DESCRIPTION
**Pull request by NetStriKerX**

### CODE REVIEW

**Comment by WatcharitNew** [comment]
Please add description about what you did on each PR

**Comment by WatcharitNew** [comment]
IMO, this PR can be grouped with controller and view

File: db/migrate/20230720080631_create_books.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+      t.string :name
```
Comment (kanatsanan6) [inline db/migrate/20230720080631_create_books.rb:4]: we can also add nullable constraint to each attributes if we don't want them to be nullable

```suggestion
      t.string :name, null: false
```

**Review by kanatsanan6 — APPROVED** [review]
Agree with New. It is good to keep our PR small, but I think it too small for this one kub

**Pull request closed without merging**